packages/cli/src/commands/buildHandler.js

```diff
@@ -89,7 +89,7 @@ export const handler = async ({
       // Vite handles this internally
       title: 'Cleaning Web...',
       task: () => {
-        rimraf.sync(rwjsPaths.web.dist)
+        rimraf(rwjsPaths.web.dist)
```

Comment:
Trying this locally and don't see it working. It may be that webpack is also removing dist before generating it, cause I do it see it being deleted, but not by this line (setting and stepping over breakpoints). But if I change this to return, it looks like it works. The listr docs aren't too great so I don't have a source:

```suggestion
        return rimraf(rwjsPaths.web.dist)
```